src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx

```diff
@@ -95,7 +95,7 @@ export class ProsemirrorAdapter {
         ];
     }
 
-    private async initializeTextEditor() {
+    initializeTextEditor = async () => {
```

Comment:
Asch… I missed that there was a `private` missing here. I'm adding it in another PR 🤦‍♂️

Reply:
Well spotted! Yeah it should be private : (